website/blog/2023-01-18-release-0.11.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Release Notes - Podman Desktop 0.11
+description: Podman Desktop 0.11 has been released! 
+slug: podman-desktop-release-0.11
+authors: [slemeur]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+This release note covers Podman Desktop 0.11 release changes.
+
+- **Air-Gapped Installation**: New all-in-one binaries for air-gapped installation.
+- **Feedback**: Submit feedback directly from Podman Desktop.
+- **Docker Compatibility Mode**: Information about the Docker compatibility mode.
+- **Proxy Setting**: Toggle on/off the proxy setting.
+- **Deploy to Kubernetes**: Select the namespace to deploy to Kubernetes.
+- **Registry Configuration**: Simplified registry configuration for well known registries.
+- **UX/UI Improvements**: View pod's container logs, better visual feedback, configurable editor font size, and more.
+
+Experimental Capability:
+- **Kind Support**: Create and manage Kind clusters in Podman.
+
+Podman Desktop 0.11 is now available. [Click here to download it](/downloads)! 
+
+<!--truncate-->
+_________________
+
+## Release Details
+
+### Air-Gapped installation [#1104](https://github.com/containers/podman-desktop/pull/1104) [#1120](https://github.com/containers/podman-desktop/pull/1120)
+
+For users who are not able to connect to the internet, a new all-in-one (airgap) binary that includes the Podman Desktop application and the Podman binaries is now available for both Mac and Windows. When initializing a new Podman machine using all-in-one binaries, it'll use the embedded files and not grab them from internet: 
+* on Mac, it embeds the qemu FCOS binaries.
+* on Windows, it embeds the WSL binary.
+
+To differentiate the air-gapped binaries from the regular ones because they're containing 'airgap' in their artifactname and are bigger.
```

Comment:
Missing space in artifactname and worded a little odd. Maybe something like "Air-gapped binaries are bigger than the regular ones, so the artifact names contain 'airgap' to differentiate them."?